modules/op-installing-pipelines-operator-in-web-console.adoc

```diff
@@ -41,12 +41,12 @@ The supported profiles are:
 
 .. Select an *Update Channel*.
 
-*** The `pipelines-<version>` channel is the default channel to install the {pipelines-title} Operator. For example, the default channel to install the {pipelines-title} Operator version `1.7` is `pipelines-1.7`.
+*** The `pipelines-<version>` channel is the default channel to install the {pipelines-title} Operator. For example, the default channel to install the {pipelines-title} Operator version `1.8` is `pipelines-1.8`.
```

Comment:
Now the default channel is `latest` (https://issues.redhat.com/browse/SRVKP-2412)
cc; @vdemeester